client/src/pages/Notifications/Notifications.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useEffect } from 'react';
+import Paper from '@material-ui/core/Paper';
+import Pagination from '@material-ui/lab/Pagination';
+import Typography from '@material-ui/core/Typography';
+import Box from '@material-ui/core/Box';
+import useStyles from './NotificationStyles/Notifications';
+import NotificationMessages from '../../components/Notification/NotificationMessages';
+import { getNotifications, updReadNotifications, getCount } from '../../helpers/APICalls/notifications';
+import { Notification } from '../../interface/Notifications';
+
+export default function NotificationPopover(): JSX.Element {
+  const classes = useStyles();
+  const returnNotificationType = (res: any): Notification[] => res.notifications;
+  const messages: Notification[] = [];
+  const [messagesToShow, setmessagesToShow] = React.useState(messages);
+  const [page, setPage] = React.useState(1);
+  const [count, setCount] = React.useState(1);
```

Comment:
Include `useState` in your `import` and keep the code cleaner

Reply:
done